src/main/fish/payara/server/PayaraInstanceProvider.ts

```diff
@@ -43,14 +44,23 @@ export class PayaraInstanceProvider {
         );
     }
 
-    getserversConfig(context: vscode.ExtensionContext): string {
+    private getServersConfig(context: vscode.ExtensionContext): string {
         let storagePath: string;
         if (!context.storagePath) {
             storagePath = path.resolve(os.tmpdir(), `payara_vscode`);
```

Comment:
`temp` dir is used only in case of `ExtensionContext.storagePath` value is not defined.

Reply:
Do you know why this would not be defined? It's undefined on my machine but I have a configuration directory and other plugins have persistent configuration files

Reply:
`ExtensionContext.storagePath` is undefined, In case of workspace folder is not opened.